src/longbow/model/command.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import time
+import sys
+
+import pickle
+
+import click
+import click_log
+
+import multiprocessing as mp
+
+from ..utils import model as LongbowModel
+from ..utils.model import LibraryModel
+
+from ..utils import cli_utils
+
+logging.basicConfig(stream=sys.stderr)
+logger = logging.getLogger("model")
+click_log.basic_config(logger)
+
+
+@click.command(name="model")
+@click_log.simple_verbosity_option(logger)
+@click.option(
+    "-l",
+    "--list-models",
+    cls=cli_utils.MutuallyExclusiveOption,
+    mutually_exclusive=["dump"],
+    required=False,
+    is_flag=True,
+    default=False,
+    help="List the names of all models supported natively by this version of Longbow."
+)
+@click.option(
+    "-d",
+    "--dump",
+    cls=cli_utils.MutuallyExclusiveOption,
+    mutually_exclusive=["list_models"],
+    required=False,
+    type=str,
+    help="Dump the details of a given model.  "
+         "This command creates a set of files corresponding to the given model "
+         "including: json representation, dot file representations, transmission matrix, state emission json file."
+)
+def main(list_models, dump):
+    """Get information about built-in Longbow models."""
+
+    logger.info("Invoked via: longbow %s", " ".join(sys.argv[1:]))
+
+    if list_models or (not list_models and dump is None):
+        print("Longbow includes the following models:")
+        model_info_list = []
+        col_widths = [0, 0, 0]
+        for k, v in LibraryModel.pre_configured_models.items():
+            model_info_list.append((k, v["version"], v["description"]))
+
+            # Make sure we can print things nice and tidy:
+            if len(k) > col_widths[0]:
```

Comment:
I'm fine with this, but in case you're curious, there do seem to be some packages that format tables nicely and automatically: https://stackoverflow.com/questions/9535954/printing-lists-as-tabular-data .

Reply:
Good point.  In the future if we end up needing another pretty-printed table, we should refactor this.